collections/join_to_string.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Options for joinToString
+ */
+export type JoinToStringOptions = {
+  separator?: string;
+  prefix?: string;
+  suffix?: string;
+};
+
+/**
+ * Transforms the elements in the given array to strings using the given selector.
+ * Joins the produced strings into one using the given separator and applying the given prefix and suffix to the whole string afterwards.
+ * Returns the resulting string.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { joinToString } from "./join_to_string.ts";
```

Comment:
See #1227 , I think we should use the same `import` as in the `README` here

Reply:
oh, nice. that was something that bothered me a lot. I didn't know somebody fixed this in another PR.